### PR TITLE
Add bicep template for create vm and configure protection template

### PR DESCRIPTION
**Pull request by nilaydshah**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add bicep template for create vm and configure protection template
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@v-amallick - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.recoveryservices/recovery-services-create-vm-and-configure-backup/main.bicep [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+param adminPassword string
+
+@description('Specifies the unique DNS Name for the Public IP used to access the Virtual Machine.')
+param dnsLabelPrefix string
+
+@description('Virtual machine size.')
+param vmSize string = 'Standard_A2'
+
+@description('Specifies the Windows version for the VM. This will pick a fully patched image of this given Windows version. Allowed values: 2008-R2-SP1, 2012-Datacenter, 2012-R2-Datacenter, 2016-Nano-Server, 2016-Datacenter-with-Containers, 2016-Datacenter.')
+@allowed([
```
[the remark this answers is not in the document]
Comment (nilaydshah) [reply quickstarts/microsoft.recoveryservices/recovery-services-create-vm-and-configure-backup/main.bicep:22]: Addressed review comment.

**Comment by azure-quickstarts** [comment]
@v-amallick - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Review by v-amallick — APPROVED** [review]

**Review by v-amallick — APPROVED** [review]

**Comment by azure-quickstarts** [comment]
@v-amallick - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Review by v-amallick — APPROVED** [review]

**Comment by azure-quickstarts** [comment]
@v-amallick - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@v-amallick - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Pull request merged**